modules/programs/qutebrowser.nix

```diff
@@ -260,22 +260,31 @@ in {
     };
   };
 
-  config = mkIf cfg.enable {
-    home.packages = [ cfg.package ];
-
-    xdg.configFile."qutebrowser/config.py".text = concatStringsSep "\n" ([ ]
-      ++ [
-        "${if cfg.loadAutoconfig then
-          "config.load_autoconfig()"
-        else
-          "config.load_autoconfig(False)"}"
-      ] ++ mapAttrsToList (formatLine "c.") cfg.settings
+  config = let
+    qutebrowserConfig = concatStringsSep "\n" ([
+      (if cfg.loadAutoconfig then
+        "config.load_autoconfig()"
+      else
+        "config.load_autoconfig(False)")
+    ] ++ mapAttrsToList (formatLine "c.") cfg.settings
       ++ mapAttrsToList (formatDictLine "c.aliases") cfg.aliases
       ++ mapAttrsToList (formatDictLine "c.url.searchengines") cfg.searchEngines
       ++ mapAttrsToList (formatDictLine "c.bindings.key_mappings")
       cfg.keyMappings
       ++ optional (!cfg.enableDefaultBindings) "c.bindings.default = {}"
       ++ mapAttrsToList formatKeyBindings cfg.keyBindings
       ++ optional (cfg.extraConfig != "") cfg.extraConfig);
+  in mkIf cfg.enable {
+    home.packages = [ cfg.package ];
+
+    home.file.".qutebrowser/config.py" =
+      mkIf (cfg.settings != { } && pkgs.stdenv.hostPlatform.isDarwin) {
```

Comment:
On second thought, might be because of `cfg.settings != { }` but I'm not sure, will test when I'm free again.